requirements-dev.txt

```diff
@@ -0,0 +1,20 @@
+black
+check-manifest
+doctr
```

Comment:
All the development dependencies here are used in the pre-commit or other checks except for `doctr`, this one will be used later to publish the docs.